src/main/java/no/nav/familie/integrasjoner/geografisktilknytning/GeografiskTilknytningDto.kt

```diff
@@ -0,0 +1,21 @@
+package no.nav.familie.integrasjoner.geografisktilknytning
+
+data class PdlHentGeografiskTilknytning(val hentGeografiskTilknytning: GeografiskTilknytningDto)
```

Comment:
Mener å huske at denne kan bli `null` i gitte tilfeller - så sleng på `?` og sjekk for null -> hvis null så bør vi kaste feil. 

Typisk http-feil som gjør at kallet aldri kommer frem til "koden" i pdl, men stoppes i jersey/spring-rammeverket.